ui/src/overridden/frontsite/Document/DocumentDetails/DocumentItemBody.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { DocumentItemBody } from "@inveniosoftware/react-invenio-app-ils";
+import _get from "lodash/get";
+import { parametrize } from "react-overridable";
+import { Table } from "semantic-ui-react";
+
+export const DocumentItemTableHeader = () => {
+  return (
+    <Table.Row data-test="header">
+      <Table.HeaderCell>Barcode</Table.HeaderCell>
+      <Table.HeaderCell>Shelf</Table.HeaderCell>
+      <Table.HeaderCell>Call Number</Table.HeaderCell>
+      <Table.HeaderCell>Status</Table.HeaderCell>
+      <Table.HeaderCell>Medium</Table.HeaderCell>
+      <Table.HeaderCell>Loan restriction</Table.HeaderCell>
+    </Table.Row>
+  );
+};
+
+function renderShelflink(item) {
+  const shelfNumber = _get(item, "shelf");
+  if (shelfNumber === undefined) return;
+  const shelfLink = `https://maps.web.cern.ch/?n=['SHELF ${shelfNumber}']`;
```

Comment:
please move it to the js config, otherwise it is hard to search for

Reply:
Should I move it to [utils](https://github.com/CERNDocumentServer/cds-ils/blob/master/ui/src/overridden/frontsite/utils.js)?
We aren't overriding it so I feel config might not be the correct place. WDYT?